go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/yohcop/openid-go
+module github.com/Schrodinger-Box/openid-go
```

Comment:
This seems like a mistake.

Reply:
Sorry, my mistake. I committed that on master branch for debug purposes. I will create a new branch and submit a new PR.